### PR TITLE
Don't start the build if package with this NVR already has been built

### PR DESCRIPTION
**Pull request by vrutkovs**
See also https://github.com/projectatomic/atomic-reactor/pull/485 - a similar check in atomic-reactor

### CODE REVIEW

**Comment by twaugh** [comment]
Looking in more detail at this I think we might be using the wrong name in atomic reactor. We ought to be using the BZComponent label instead of the Name label.

**Comment by twaugh** [comment]
Compare with master branch to see how this was did prior to CG. We should do the nvr check in the same place we used to initiate the Build object.

File: koji_containerbuild/plugins/builder_containerbuild.py [hunk 1]
```diff
@@ -619,6 +624,16 @@ def handler(self, src, target, opts=None):
                                       "target": target}
             if not SCM.is_scm_url(src):
                 raise koji.BuildError('Invalid source specification: %s' % src)
+            # Scratch builds shouldn't be checked for nvr
+            if not self.opts.get('scratch'):
+                expected_nvr = labels_wrapper.get_expected_nvr()
+                try:
+                    build_id = self.xmlrpc.getBuild(expected_nvr)['id']
+                except:
+                    self.logger.info("No build for %s found" % expected_nvr)
```
Comment (twaugh) [inline koji_containerbuild/plugins/builder_containerbuild.py]: Could change "%" to "," to use lazy string formatting provided by logging functions.

Comment (vrutkovs) [inline koji_containerbuild/plugins/builder_containerbuild.py]: Will do

**Comment by twaugh** [comment]
LGTM

**Comment by breillyrh** [comment]
Looks good to me as well, I'll merge.

**Pull request merged**